app/models/mixins/supports_feature_mixin.rb

```diff
@@ -73,6 +73,7 @@ module SupportsFeatureMixin
     :backup_restore                      => 'CloudVolume backup restore',
     :cinder_service                      => 'Cinder storage service',
     :cinder_volume_types                 => 'Cinder volume types',
+    :conversion                          => 'Conversion host support',
```

Comment:
`:conversion_host` would be better I think, since we don't want to confuse `resource.supports_conversion?` which sounds like you are asking if it can be converted, not if you are able to make it a conversion_host.

Reply:
yeah, need to change the name.

This seems gonna be a difficult to distinguish between `conversion_host capable` and `conversion_host ready`

Reply:
Alright, had that originally, was just thinking `supports_conversion` sounded better, but I can understand the possible confusion.

Reply:
Yeah I think just `supports_conversion_host?` is fine, conversion_host ready can be done by checking e.g. `vm.conversion_host.present?`